lib/prometheus-metrics.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
+class PrometheusMetrics {
+  constructor (circuit) {
+    this.circuit = circuit;
+    // this.client = require('prom-client');
```

Comment:
Not clear why this isn't working - I think it should. Then, subsequently, I suggest that you initialize an actual client instance here instead of in `collect()`.

```suggestion
    this.client = require('prom-client');
    // Use the circuit breaker's name as a prefix
    // Keep a handle on the interval so we can clear it later
    this.interval = client.collectDefaultMetrics({ prefix: `${circuit.name}_` });
```

The `collect()` method below can probably just all be added to this constructor, since I don't think we want to expose `collect()` externally.